test/apitest.py

```diff
@@ -898,15 +960,16 @@ def test_routes_all_verbs(self):
         # Patch add
         r = self.patch_config_vrouter_vrf_id_routes("vnet-guid-1", routes)
         self.assertEqual(r.status_code, 204)
-        self.check_routes_exist_in_db(1, routes)
+        self.check_routes_exist_in_tun_tb(1, routes)
         r = self.patch_config_vrouter_vrf_id_routes("vnet-guid-2", routes)
         self.assertEqual(r.status_code, 204)
-        self.check_routes_exist_in_db(2, routes)
+        self.check_routes_exist_in_tun_tb(2, routes)
         # Patch delete
 
         # Get all
         for route in routes:
              del route['cmd']
+        routes.append({'nexthop': '', 'ip_prefix': '10.1.1.0/24', 'ifname': 'Vlan2'})
```

Comment:
Backend might behave differently if `ipaddres `is empty string. So its better to remove the attribute from route structure if not provided.

Reply:
yes, I will add a if clause to check

Reply:
line 230 in models.go